tests/installation/bootloader_svirt.pm

```diff
@@ -272,38 +272,40 @@ sub run {
         $svirt->suspend;
         my $cmdline = '';
         $cmdline .= 'textmode=1 '                         if check_var('VIDEOMODE', 'text');
-        $cmdline .= 'rescue=1 '                           if is_installcheck || is_rescuesystem;          # rescue mode
+        $cmdline .= 'rescue=1 '                           if is_installcheck || is_rescuesystem;                         # rescue mode
         $cmdline .= get_var('EXTRABOOTPARAMS') . ' '      if get_var('EXTRABOOTPARAMS');
-        $cmdline .= registration_bootloader_cmdline . ' ' if check_var('SCC_REGISTER', 'installation');
+        $cmdline .= registration_bootloader_cmdline . ' ' if check_var('SCC_REGISTER', 'installation') and (!is_jeos);
```

Comment:
Have you considered to unset this setting for jeos? You could set `+SCC_REGISTER=never` in the job group and it will take precedence.

Reply:
I couldn't find it in [variables.md](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md) hoping that someone will point it out in the PR. :) Very much appreciated, thanks! I will update the test suites.

Reply:
Yeah, that variable is weird, so basically all values except `installation` and `yast` will do the trick.

Reply:
And it works well for RT (or other add-on) when it comes to post install registration or for installation scenarios. 

On the other hand there is a bit of a gap in case of JeOS. It doesn't have YaST2, it is already pre-installed and registration is done by suseconnect. 

Nevertheless `+SCC_REGISTER=never` does the trick

Reply:
You just have to not define SCC_REGISTER, but yes any other string than `installation` and `yast` will work

Reply:
> You just have to not define SCC_REGISTER, but yes any other string than `installation` and `yast` will work

It wasn't defined and it appeared in vars.json.

Reply:
it is defined in main.pm :P